roles/edpm_libvirt/tasks/virsh-secret.yml

```diff
@@ -35,7 +35,7 @@
   ansible.builtin.command: "podman exec libvirt_virtqemud bash -c 'virsh secret-set-value $FSID --base64 $KEY'"
   environment:
     FSID: "{{ fsid }}"
-    KEY: "{{ cephx_key.stdout }}"
+    KEY: "{{ cephx_key.stdout | regex_replace('\"', '') }}"
```

Comment:
nit: it would have been nice to also handle single quotes "'" as well as  well as double quotes '"'